### DISCUSSION TITLE
Help ! My repo is not detecting any language ?

### DISCUSSION THREAD

**Discussion by SidPaliwal** [github-linguist/linguist#5278] [Classification] [answered]
My repo [https://github.com/SidPaliwal/NewsApp](url) is not showing any language ? Can anyone please resolve this ?

**Comment by lildude** [comment, marked as answer]
🤔 looks like there might have been a problem when you pushed your changes two months ago that resulted in the background job not running or failing. Looks to be working now though.

In future, you can trigger this by pushing another change to your repo.

**Reply by JohnLulzh2002** [reply]
I encountered this problem [here](https://github.com/JohnLulzh2002/LZH-AI-study), and I have pushed more than 10 times since last month, but it remains unchanged.
I wonder if I can trigger the 'background job' you mentioned manually.

**Reply by lildude** [reply]
@JohnLulzh2002 your repo is _not_ experiencing the same problem. It is currently reporting two languages: "Rich Text Format" and "Other". The "Other" covers all the other languages because your repo has so much "Rich Text Format" by bytes of code thanks to this [one file](https://github.com/JohnLulzh2002/LZH-AI-study/blob/master/PythonCrashCourse/4.4.3-WhyCpoy.rtf) being 18.3MB which is significantly larger than all the other code in your repo combined.

Either remove the file or implement an override to ignore it and you'll then see Jupyter and Python appear in the list of languages.
